src/sagemaker/jumpstart/constants.py

```diff
@@ -70,6 +70,11 @@
             gated_content_bucket="jumpstart-private-cache-prod-eu-central-1",
             neo_content_bucket="sagemaker-sd-models-prod-eu-central-1",
         ),
+        JumpStartLaunchedRegionInfo(
+            region_name="eu-central-2",
+            content_bucket="jumpstart-cache-prod-eu-central-2",
+            gated_content_bucket="jumpstart-private-cache-prod-eu-central-2",
+        ),
```

Comment:
Is the neo bucket not needed for this region?

Reply:
Added to description: Adding JS bucket info. I verified through the Neo RIP service that their bucket is not available as of yet.

Reply:
Then should you remove the neo bucket?